simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -298,13 +298,12 @@ function simple_ldap_user_import($form, &$form_state) {
   
   $new_names = $users;
   $sql = "select name from {users} where name= :name";
-  foreach ($new_names as $item){
-     $result = db_query($sql, array(':name'=>$item));
-     if ($result) {
-        while ($row=$result->fetchAssoc()) {
-            // Filter users fetched from ldap database and mysql database
-
-            unset ($new_names[strtoupper($row['name'])]);
+  foreach ($new_names as $item) {
+    $result = db_query($sql, array(':name'=>$item));
```

Comment:
Include spaces around the `=>` operator.  See: https://www.drupal.org/docs/develop/standards/coding-standards#arrays